src/main/java/org/gridsuite/shortcircuit/server/utils/FaultResultSpecificationBuilder.java

```diff
@@ -28,8 +32,27 @@ public static Specification<FaultResultEntity> resultUuidEquals(UUID value) {
     }
 
     public static Specification<FaultResultEntity> buildSpecification(UUID resultUuid, List<ResourceFilter> resourceFilters) {
-        Specification<FaultResultEntity> specification = Specification.where(resultUuidEquals(resultUuid));
-        return SpecificationUtils.appendFiltersToSpecification(specification, resourceFilters);
+        return (final Root<FaultResultEntity> root, final CriteriaQuery<?> query, final CriteriaBuilder builder) -> {
```

Comment:
I personnaly prefer how it's done on SA : https://github.com/gridsuite/security-analysis-server/blob/555a4951b760b5cc7af634863c52ffa314e9e778/src/main/java/org/gridsuite/securityanalysis/server/repositories/specifications/AbstractCommonSpecificationBuilder.java#L31
Easier to read and understand IMO

Reply:
I copied the SA specification classes. It is indeed easier to read and understand and it is better to have similar systems anyway.

Reply:
I copied the SA specification classes. It is indeed easier to read and understand and it is better to have similar systems across the microservices anyway.